src/transformers/models/vit/convert_vit_timm_to_pytorch.py

```diff
@@ -219,8 +198,8 @@ def convert_vit_checkpoint(vit_name, pytorch_dump_folder_path):
 
     if base_model:
         timm_pooled_output = timm_model.forward_features(pixel_values)
-        assert timm_pooled_output.shape == outputs.pooler_output.shape
-        assert torch.allclose(timm_pooled_output, outputs.pooler_output, atol=1e-3)
+        assert timm_pooled_output.shape == outputs.last_hidden_state.shape
+        assert torch.allclose(timm_pooled_output, outputs.last_hidden_state, atol=1e-1)
```

Comment:
not sure if the last hidden states are always gonna have the same shape as the pooled outputs?

Reply:
The 'timm_pooled_output' is the output of the forward_features method ([L656](https://github.com/huggingface/pytorch-image-models/blob/e1e7cf5275600ea74d7b6ed80cc105701bddefdb/timm/models/vision_transformer.py#L656)) which does not include a pooling layer, so I don't know why it's called a pooled output. I might be wrong though

Reply:
that is correct, forward_features() is unpooled output